Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -3119,12 +3119,28 @@ void CGContextDrawTiledImage(CGContextRef context, CGRect rect, CGImageRef image
 #pragma endregion
 
 #pragma region Drawing Operations - Gradient + Shading
+
+static bool _isFloatEqual(const float a, const float b) {
```

Comment:
nit: not so sure this should be declaring them equal, maybe __CloseEnough or something like that but... more reasonable.  Also they're not references, why const?  And CGFloat?